templates/miq-template.yaml

```diff
@@ -357,7 +357,7 @@ objects:
       description: Defines how to deploy the ManageIQ appliance
   spec:
     serviceName: "${NAME}"
-    replicas: "${APPLICATION_REPLICA_COUNT}"
+    replicas: "${{APPLICATION_REPLICA_COUNT}}"
```

Comment:
Can these be two separate PR's?

Reply:
This seems to have been fine outside of the context of upgrade so I think it really only applies when talking about upgrading, but I can extract it if you feel strongly about it.